core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -79,7 +79,7 @@ oppia.factory('playerTranscriptService', ['$log', function($log) {
 
       lastCard.destStateName = newDestStateName;
     },
-    addNewAnswer: function(answer) {
+    addNewAnswer: function(answer, isHintRequest) {
```

Comment:
This seems a bit clumsy, since no answer is actually being submitted. Perhaps have a new method called addHintRequest() or requestHint()?